client/CMakeLists.txt

```diff
@@ -283,6 +283,11 @@ target_link_libraries(vcmiclient PRIVATE
 )
 
 if(ffmpeg_LIBRARIES)
+	if(USING_CONAN AND MINGW)
+		target_link_options(vcmiclient PRIVATE
```

Comment:
Can you explain (in comment in cmake) what is this and why is it needed?

Reply:
Removed this and replaced by another, more correct workaround.